### ISSUE TITLE
Beta is not saving scan files

### ISSUE THREAD

**Issue by minimaker-nl** [OpenScanEu/OpenScan2#3]
I'm testing the Beta software on a Pi3B using the new arducam camera. In the first tests scanning and saving files worked without problems. In the last tests it's not saving files any more.

I don't know if it's related but this was after I removed all scanfiles with the delete all button. Also, I accidentally clicked the opencloud refresh button.

My next step is that I will reflash the SD and try again. But if there are any tests I should do before that, please let me know.

**Comment by minimaker-nl** [comment]
To add, I flashed another SD and saving files does work with that SD. I'm keeping the problem version SD around for now in case you want me to test something.

**Comment by minimaker-nl** [comment]
I just tried to recreate the problem on the new installation but have not yet succeeded. I did notice a fe oddities:
- Start situation: 4 files in Files&Cloud list
- Selected one file and clicked the delete set button - no effect. Tried on another set - no effect. The delete warning does appear though.
- Went back to scan window and returned to Files&Cloud. "Deleted" sets are still listed.
- Downloaded one of the "deleted" sets - successful. The set still exists.
- Delete all - removes all sets correctly.
- Created new scan - set as listed in Files & Cloud.
- Delete all, click refresh (no token installed) - no token warning. Status stays on "refreshing".
- Create new set: gets listed in Files&Cloud.

Perhaps the problem reported before is not that much that no new sets are added to the list but that there is an issue in the update of the listing of sets on "Files  & Cloud".
